geonode/templates/base.html

```diff
@@ -151,6 +144,28 @@
       <p>Please use a <strong>modern browser</strong> like Mozilla Firefox, Google Chrome or Safari.</p>
     </div>
```

Comment:
I have examined this code. I think it specifically targets at the old IE browser. The code and explanation is [here](https://github.com/afarkas/html5shiv)